config/deploy.js

```diff
@@ -101,6 +101,13 @@ function deployBuild(url, folder) {
         let next_path = path.resolve(folder);
         shell.cd(next_path);
         const start = Date.now();
+        // Copy previously bundled client resources to deploy along with newly generated resources
+        if (url === EDITOR_URL) {
+            shell.exec('git clone ' + url + ' current_build');
+            shell.exec('cp -f current_build/*.js current_build/*.css .');
```

Comment:
Actually it makes sense to use "cp -n" here instead of "cp -f". If WebPack for some reason has a name collision with previous file contents, we will prefer the newer version.